src/olympia/addons/tests/test_views.py

```diff
@@ -1704,6 +1704,8 @@ def test_get_addon_not_found(self):
 
 
 class TestAddonViewSetDetail(AddonAndVersionViewSetDetailMixin, TestCase):
+    client_class = APITestClient
```

Comment:
We can, I almost did this, but then we'd also need an `APIESTestCase`... Not sure what's the best path. 

Note that ultimately, it's not *required* it's just nice to have because it does json by default so you don't have to set content type and pass json data yourself.